common/pb/event_leave.proto

```diff
@@ -5,4 +5,11 @@ message Event_Leave {
     extend GameEvent {
         optional Event_Leave ext = 1001;
     }
+    enum LeaveReason {
+        OTHER = 1;
+        USER_KICKED = 2;
+        USER_LEFT = 3;
+        USER_DISCONNECTED = 4;
```

Comment:
When a user closes the game it'll show "left"?
When a user closes the app it'll show "left" or "disconnect"?
When a user restarts his PC or his Internet connection drops, what will be shown?